### PR TITLE
Fix a couple of typos in Jsrt Header comments

### PR DESCRIPTION
**Pull request by rhuanjl**
This PR fixes a few typos I noticed in the Jsrt Header comments as well as one pointed out by @RealTrisT in #5615

### CODE REVIEW

File: lib/Jsrt/ChakraCommon.h [hunk 1]
```diff
@@ -237,6 +237,7 @@ typedef unsigned short uint16_t;
         ///     Module was not yet evaluated when JsGetModuleNamespace was called.
         /// </summary>
         JsErrorModuleNotEvaluated,
+
```
Comment (dilijev) [inline lib/Jsrt/ChakraCommon.h]: why add this blank line?

Comment (rhuanjl) [reply lib/Jsrt/ChakraCommon.h]: The next line is the start of a category, all the other categories are preceded by a blank line.

Comment (dilijev) [reply lib/Jsrt/ChakraCommon.h]: SGTM

File: lib/Jsrt/ChakraCommon.h [hunk 2]
```diff
@@ -199,7 +199,7 @@ typedef unsigned short uint16_t;
         /// </summary>
         JsErrorInObjectBeforeCollectCallback,
         /// <summary>
-        ///     Object cannot be unwrapped to IInspectable pointer.
+        ///     Object cannot be unwrapped to Inspectable pointer.
```
[the remark this answers is not in the document]
Comment (jackhorton) [reply lib/Jsrt/ChakraCommon.h]: .NET and, possibly more importantly, WinRT convention. I think this has to do with JSRT Projection

**Comment by jackhorton** [comment]
Shouldn't the wiki be auto-generate-able from the header? Id imagine we already do that for the MSDN page.

File: lib/Jsrt/ChakraCommon.h [hunk 3]
```diff
@@ -217,7 +217,7 @@ typedef unsigned short uint16_t;
         /// </summary>
         JsErrorInvalidContext,
         /// <summary>
-        ///     Module evaluation is called in wrong context.
+        ///     Invalid Module HostInfoKind provided to JsSetModuleHostInfo or JsGetModuleHostInfo.
```
Comment (liminzhu) [inline lib/Jsrt/ChakraCommon.h]: I'd avoid mentioning specific APIs for future-proofing and just say `The Module HostInfoKind provided is invalid.`

**Pull request merged**